apps/designer/app/dashboard/new-project.tsx

```diff
@@ -35,15 +46,51 @@ const useNewProject = () => {
     navigate(designerPath({ projectId: data.id }));
   }, [data, navigate]);
 
-  return handleCreate;
+  return { handleChange, handleCreate };
 };
 
 export const NewProject = () => {
-  const handleCreate = useNewProject();
+  const { handleCreate, handleChange } = useNewProject();
 
   return (
-    <Button prefix={<PlusIcon />} onClick={handleCreate}>
-      New Project
-    </Button>
+    <FloatingPanelDialog.Root>
+      <FloatingPanelDialog.Trigger asChild>
+        <Button prefix={<PlusIcon />}>New Project</Button>
+      </FloatingPanelDialog.Trigger>
+      <FloatingPanelDialog.Content>
+        <FloatingPanelDialog.Description asChild>
+          <Flex
+            direction="column"
+            css={{
+              px: theme.spacing["9"],
+              py: theme.spacing["5"],
+              paddingBottom: theme.spacing["10"],
+            }}
+            gap="1"
+          >
+            <Label>Project Title</Label>
+            <form onSubmit={handleCreate}>
+              <TextField placeholder="New Project" onChange={handleChange} />
+            </form>
+          </Flex>
+        </FloatingPanelDialog.Description>
```

Comment:
Not sure `<Description>` will work well with all this stuff inside. In their own demo Radix developers wrap only a line of text inside it instead of entire dialog content

https://www.radix-ui.com/docs/primitives/components/dialog

<img width="487" alt="Screenshot 2023-01-31 at 09 36 43" src="https://user-images.githubusercontent.com/825702/215675041-52548a90-060b-42e7-a87a-58f91085061c.png">

And you don't have to always have a descriptions AFAIK.

Reply:
You are right

Reply:
just read what description is for, its accessibility feature